enterprise/backend/test/metabase_enterprise/sso/integrations/jwt_test.clj

```diff
@@ -61,14 +63,15 @@
 
 (defmacro ^:private with-jwt-default-setup [& body]
   `(mt/with-ensure-with-temp-no-transaction!
-     (disable-other-sso-types
-      (fn []
-        (with-sso-jwt-token
-          (saml-test/call-with-login-attributes-cleared!
-           (fn []
-             (call-with-default-jwt-config
-              (fn []
-                ~@body)))))))))
+     (premium-features-test/with-premium-features #{:audit-app}
```

Comment:
Audit logging was disabled inside of this macro, until I added the audit-app feature, and made `disable-other-sso-types` use `with-additional-premium-features`.